### PR TITLE
emqttd - Erlang MQTT Broker

### PR DESCRIPTION
**Pull request by turtleDeng**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/emqttd.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+class Emqttd < Formula
+  desc "Erlang MQTT Broker"
+  homepage "http://emqtt.io"
+  url "http://emqtt.io/emqttd-macosx-2.1.0.zip"
+  sha256 "c3d213c1085e60ad767fe277635d369620903f041fadc0fb8c7630277b5d3af9"
```
Comment (dunn) [inline Formula/emqttd.rb]: SHA mismatch:
```
==> Downloading http://emqtt.io/emqttd-macosx-2.1.0.zip
Warning: Formula reports different SHA256: c3d213c1085e60ad767fe277635d369620903f041fadc0fb8c7630277b5d3af9
Downloaded to: /Users/brew/Library/Caches/Homebrew/emqttd-2.1.0.zip
SHA256: 9d25255ec3376a86b5ea7a70080b127ba385643c22d91b6f704db954627bbec5

File: Formula/emqttd.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
+class Emqttd < Formula
+  desc "Erlang MQTT Broker"
+  homepage "http://emqtt.io"
+  url "http://emqtt.io/emqttd-macosx-2.1.0.zip"
```
Comment (dunn) [inline Formula/emqttd.rb]: We need to build this from source.

File: Formula/emqttd.rb (hunk 1 above)
Comment (turtleDeng) [inline Formula/emqttd.rb]: I do not know why there will be SHA mismatch:
Http://emqtt.io/emqttd-macosx-2.1.0.zip package sha256 is c3d213c1085e60ad767fe277635d369620903f041fadc0fb8c7630277b5d3af9
I took the machine new mac
1. cp emqttd.rb  /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/
2. brew install emqttd
3. The installation was successful

**Comment by turtleDeng** [comment]
@dunn 
I newly modified my zip package
Through the 
`
wget http://emqtt.io/emqttd-macosx-2.1.zip 
openssl dgst -sha256 emqttd-macosx-2.1.zip
`

**Comment by dunn** [comment]
We still need it built from source.

**Comment by fxcoudert** [comment]
Hi @turtleDeng, and thanks for the pull request. As @dunn commented, we install software from source. I will close this for now, please reopen it with an update to the formula that builds the software from a source package.

**Pull request closed without merging**